.pre-commit-config.yaml

```diff
@@ -54,3 +54,23 @@ repos:
           docs/docs/javascripts/.*|
           docs/docs/webfonts/.*
       )$
+- repo: https://github.com/pre-commit/mirrors-prettier
+  rev: "v3.0.0-alpha.9-for-vscode"
+  hooks:
+    - id: prettier
+      files: ^src/frontend/.*\.(js|jsx|ts|tsx)$
+      additional_dependencies:
+      - "prettier@^2.4.1"
+      - "@trivago/prettier-plugin-sort-imports"
```

Comment:
Are you sure there is no additional configuration needed about the groups, I always set sort groups manually where I used this plugin.

Reply:
In my tests, it sorted without groups set but I will add the default

Reply:
Add ed